build_release.sh

```diff
@@ -17,4 +17,4 @@ unzip dist/rtg-tools*.zip
 cp rtg-tools*/RTG.jar $cwd
 popd
 
-tar -cv utils.py compare_vcf.py RTG.jar quickstart.sh VarSim.jar __init__.py liftover_restricted_vcf_map.py generate_small_test_ref.py varsim.py varsim_multi.py varsim_somatic.py varsim_validator.py | gzip > varsim-$version.tar.gz
+tar -cv build.sh utils.py compare_vcf.py RTG.jar quickstart.sh VarSim.jar __init__.py liftover_restricted_vcf_map.py generate_small_test_ref.py varsim.py varsim_multi.py varsim_somatic.py varsim_validator.py | gzip > varsim-$version.tar.gz
```

Comment:
Since you already added a build script. The build script should be invoked here. And you should omit packing `build.sh` here since this is packing the compiled files.

Reply:
Thanks. But `build.sh` is for building `python` and `java` environment on user end, so I guess we need it here?

Reply:
Got it. Thanks.